src/main/java/com/scalar/database/transaction/consensuscommit/Coordinator.java

```diff
@@ -25,7 +25,9 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-/** */
+/**
+ * A coordinator for distributed storage
```

Comment:
Can you elaborate on the explanation a little bit more ?
What does it do ?
What does it coordinate ?

Reply:
Thank you. Again, I wrote something, but please check it for correctness. As far as I can tell, it is storing snapshot ids with their state, but the `NoMutationException` in `put` is throwing me off. Are their mutations, or is putting the id with its state considered a mutation?